assets/dev/js/editor/utils/conditions.js

```diff
@@ -32,52 +28,85 @@ Conditions = function() {
 			default:
 				return leftValue === rightValue;
 		}
-	};
+	}
+
+	/**
+	 * Get Operator
+	 *
+	 * Returns the condition's comparison operator according to the structure of the condition and control values.
+	 *
+	 * @since 3.7.0
+	 *
+	 * @param conditionValue
+	 * @param isNegativeCondition
+	 * @param currentValue
+	 * @returns {string}
+	 */
+	getOperator( conditionValue, isNegativeCondition, currentValue ) {
+		let operator;
+
+		if ( Array.isArray( conditionValue ) && conditionValue.length ) {
+			operator = isNegativeCondition ? '!in' : 'in';
+		} else if ( Array.isArray( currentValue ) && currentValue.length ) {
+			operator = isNegativeCondition ? '!contains' : 'contains';
+		} else if ( isNegativeCondition ) {
+			operator = '!==';
+		}
+
+		return operator;
+	}
+
+	/**
+	 * Get Condition Value
+	 *
+	 * Retrieves a passed condition's value. Dynamic values take precedence. If there is no dynamic value, this method
+	 * checks for a regular control value.
+	 *
+	 * @since 3.7.0
+	 *
+	 * @param comparisonObject A settings object (e.g. element settings - keys and values)
+	 * @param conditionName The conditioning control's name
```

Comment:
Remove all `control` terms from the file

Reply:
Done.